src/main/webapp/resources/js/pages/projects/remote/ProjectSynchronizationSettings.jsx

```diff
@@ -0,0 +1,113 @@
+import {
+  updateRemoteProjectSyncSettings,
+  getRemoteProjectSyncSettings }
+from "../../../apis/projects/remote-projects";
+import { formatDate } from "../../../utilities/date-utilities";
+import { SyncFrequencySelect } from "../../../components/remote-api/SyncFrequencySelect";
+import { HelpPopover } from "../../../components/popovers";
+
+/**
+ * React component for render the remote project sync settings.
+ * @returns {*}
+ * @constructor
+ */
+export function ProjectSynchronizationSettings() {
+  const projectId = window.location.pathname.match(/projects\/(\d+)/)[1];
+  const syncNowEnabledStates = ["SYNCHRONIZED", "ERROR"];
+  const [remoteProjectData, setRemoteProjectData] = useState(null);
+  const [disableSyncNow, setDisableSyncNow] = useState(false);
+
+  /*
+  When this component is rendered, query the api for the specific settings
+  for this remote project.
+   */
+  useEffect(() => {
+    getRemoteProjectSyncSettings(projectId).then(remoteProjectSettings => {
+      setRemoteProjectData(remoteProjectSettings);
+    }).catch((message) => {
+      notification.error({ message });
+    });
+  }, []);
+
+  const syncSettings = remoteProjectData === null ? [] : [
+      {
+        title: i18n("ProjectRemoteSettings.lastSync"),
+        desc: (<div>
+          <span>{formatDate({ date: remoteProjectData.lastUpdate })}</span>
+          <br />
+          <Button type="primary"
+                  onClick={() => updateSyncSettings({ forceSync: true })}
+                  disabled={(disableSyncNow ? true : false) ||
+                            (syncNowEnabledStates.includes(remoteProjectData.remoteStatus.syncStatus) ? false : true)}
+                  className="t-sync-now-btn"
+          >
+            {i18n("ProjectRemoteSettings.syncNow")}
+          </Button>
+        </div>)
+      },
+      {
+        title: i18n("ProjectRemoteSettings.remoteConnection"),
+        desc: (<span>{remoteProjectData.remoteAPI.label}<RemoteApiStatus
+          key="status"
+          api={{ id: remoteProjectData.remoteAPI.identifier }}
+        /></span>)
+      },
+      {
+        title: <span>
+                <span>{i18n("ProjectRemoteSettings.syncFrequency")}</span>
+                <HelpPopover
+                  content={<div>{i18n("SyncFrequencySelect.frequency.help")}</div>}
+                />
+               </span>,
+        desc: (
+          <Form initialValues={{
+            frequency: remoteProjectData.projectSyncFrequencies.indexOf(remoteProjectData.projectSyncFrequency),
+          }}>
+            <SyncFrequencySelect
+              onChange={(e) => updateSyncSettings({ projectSyncFrequency: remoteProjectData.projectSyncFrequencies[e] })}
+              labelRequired={false}
+            />
+          </Form>
+        )
+      },
+      {
+        title: i18n("ProjectRemoteSettings.syncUser"),
+        desc: (<div>
+          <span>{remoteProjectData.syncUser.label}</span>
+          <br />
```

Comment:
You only need the `br` tag here because you used a `span` above which is an inline element.  Make it a block element `div` is the most basic, but you could you semantically appropriate one like a heading or paragraph.

Reply:
Updated in 910177cfbe369c72a9c125601432c54076e13bc2